RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronIDProducer.cc

```diff
@@ -149,9 +191,21 @@ void LowPtGsfElectronIDProducer::produce(edm::StreamID, edm::Event& event, const
       if (gsf.isNull()) {
         continue;
       }
+
+      // Extract Track
+      const reco::Track* trk = nullptr;
+      if (useGsfToTrack_) {
+        trk = (*gsf2trk)[gsf].get();
```

Comment:
the master version has more checks before doing `.get`, in particular `isAvailable` is present.
@makortel does the `Ref::get` throw if the product is not available or will it return a nullptr?

Reply:
It throws an exception if the product is not available.

Reply:
@bainbrid 
please update this with isAvailable to match the master version so that we don't potentially get more crashes here than would be expected in 12X.

Reply:
Done. I add the check within [here](https://github.com/cms-sw/cmssw/pull/35341/files?file-filters%5B%5D=.cc#diff-e7fc6e2e8520a28d35f7f2df77a18f0dd885d69e2bcf20f530614935c99b445bR197-R199) and I do not (on purpose touch the lines [here](https://github.com/cms-sw/cmssw/pull/35341/files?file-filters%5B%5D=.cc#diff-e7fc6e2e8520a28d35f7f2df77a18f0dd885d69e2bcf20f530614935c99b445bR186-R193))